src/components/capabilities/capabilities.stories.js

```diff
@@ -49,7 +49,32 @@ const Template = () => `
 
     customElements.define("app-card", Card);
     </pre>
+  </div>
```

Comment:
looks like the formatting is off a bit here

Reply:
Good catch. Updated.